content/en/docs/contribute/style/style-guide.md

```diff
@@ -115,45 +121,54 @@ represents.
+For declarative management, use <code>kubectl apply</code>. | For declarative management, use "kubectl apply".
+Enclose code samples with triple backticks. <code>(```)</code> | Enclose code samples with any other syntax.
+Use single backticks to enclose inline code. For example, `var example = true`. | Use two asterisks (**) or an underscore (_) to enclose inline code. For example, **var example = true**.
+Use triple backticks before and after a multi-line block of code for fenced code blocks. | se multi-line blocks of code to create diagrams, flowcharts, or other illustrations.
+Use meaningful variable names that have a context. | Use variable names such as 'foo','bar', and 'baz' that are not meaningful and lack context.
+Remove trailing spaces in the code. | Add trailing spaces in the code, where these are important, because the screen reader will read out the spaces as well. 
+{{< /table >}}
 
 {{< note >}} 
 The website supports syntax highlighting for code samples, but specifying a language is optional. Syntax highlighting in the code block should conform to the [contrast guidelines.](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=141%2C143#contrast-minimum)
 {{< /note >}}
 
 ### Use code style for object field names
 
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>Set the value of the <code>replicas</code> field in the configuration file.</td><td>Set the value of the "replicas" field in the configuration file.</td></tr>
-  <tr><td>The value of the <code>exec</code> field is an ExecAction object.</td><td>The value of the "exec" field is an ExecAction object.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Use code style for object field names" >}}
+Do | Don't
+:--| :-----
+Set the value of the <code>replicas</code> field in the configuration file. | Set the value of the "replicas" field in the configuration file.
+The value of the <code>exec</code> field is an ExecAction object. | The value of the "exec" field is an ExecAction object.
+{{< /table >}}
 
 ### Use normal style for string and integer field values
 
 For field values of type string or integer, use normal style without quotation marks.
 
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>Set the value of <code>imagePullPolicy</code> to Always.</td><td>Set the value of <code>imagePullPolicy</code> to "Always".</td></tr>
-  <tr><td>Set the value of <code>image</code> to nginx:1.8.</td><td>Set the value of <code>image</code> to <code>nginx:1.8</code>.</td></tr>
-  <tr><td>Set the value of the <code>replicas</code> field to 2.</td><td>Set the value of the <code>replicas</code> field to <code>2</code>.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Use normal style for string and integer field values" >}}
+Do | Don't
+:--| :-----
+Set the value of <code>imagePullPolicy</code> to Always. | Set the value of <code>imagePullPolicy</code> to "Always".
```

Comment:
Please replace the \<code\> tags with single backticks

Reply:
Replaced <code> tags with backticks.